src/cljs/rems/search.cljs

```diff
@@ -70,49 +70,55 @@
 
 (defn search-field [{:keys [id on-search searching?]}]
   (let [input-value (r/atom "")
-        input-element (atom nil)]
+        input-element (atom nil)
+        collapse-id "application-search-tips-collapse"]
     (fn [{:keys [id on-search searching?]}]
-      [:div.search-field
-       [:label.mr-1 {:for id}
-        (text :t.search/search)]
+      [:<>
+       [:div.search-field
+        [:label.mr-1 {:for id}
+         (text :t.search/search)]
 
-       [:div.input-group.mr-2
-        [:input.form-control
-         {:id id
-          :type :text
-          :value @input-value
-          :ref (fn [element]
-                 (reset! input-element element))
-          :on-change (fn [event]
-                       (let [value (-> event .-target .-value)]
-                         (reset! input-value value)
-                         (on-search value)))}]
+        [:div.input-group.mr-2.w-50
+         [:input.form-control
+          {:id id
+           :type :text
+           :value @input-value
+           :ref (fn [element]
+                  (reset! input-element element))
+           :on-change (fn [event]
+                        (let [value (-> event .-target .-value)]
+                          (reset! input-value value)
+                          (on-search value)))}]
 
-        (when-not (= "" @input-value)
-          [:div.input-group-append
-           [:button.btn.btn-outline-secondary
-            {:id (str id "-clear")
-             :type :button
-             :aria-label (text :t.search/clear-search)
-             ;; override the custom font-size from .btn which breaks .input-group
-             :style {:font-size "inherit"}
-             :on-click (fn []
-                         (reset! input-value "")
-                         (on-search "")
-                         (.focus @input-element))}
-            [close-symbol]]])]
-
-       (when searching?
-         [spinner/small])])))
-
-(defn application-search-tips []
```

Comment:
The same `search-field` component is also used on the catalogue page and admin pages, where the application search tips don't apply.

Reply:
right! done